frontend/src/views/applications/SelectFacilityView.vue

```diff
@@ -134,11 +158,26 @@ export default {
   },
 
   methods: {
+    updateModel(updatedModel) {
+      this.updatedOrganization = cloneDeep(updatedModel)
+    },
+    deleteDocument(documentId) {
+      this.documentsToDelete.push(documentId)
+    },
+    setDocumentsComplete(value) {
+      this.documentsComplete = value
+      this.checkFacilityComplete()
+    },
+    checkFacilityComplete() {
+      this.isSelectFacilityComplete = this.documentsComplete && this.isFormComplete
+    },
     async getOrganization() {
       try {
         this.$emit('process', true)
         this.loading = true
         this.organization = await OrganizationService.getOrganization(this.userInfo?.organizationId)
+        this.organization.uploadedDocuments = await this.getCommunitySupportDocs()
```

Comment:
not sure why but I'm unable to upload documents to CRM. (using ofmqa03)

Reply:
Files were being uploaded but a timing issue was not letting them display properly. should be fixed :)